Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+
+namespace HW_4
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            while (true)
+            {
+                ChangingArray changingArray = new ChangingArray();
+
+                Console.Write("Длинна массива:");
```

Comment:
Please, use only English language in your code. Other devs and users may not be able to understand the following